### PR TITLE
scorer test: add a test to multiply the score

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: test/command/suite/select/scorer/expression/multiplication.test [hunk 1]
```diff
@@ -0,0 +1,15 @@
+table_create Users TABLE_HASH_KEY ShortText
+column_create Users tags COLUMN_VECTOR|WITH_WEIGHT ShortText
+
+table_create Tags TABLE_HASH_KEY ShortText
+column_create Tags users_tags COLUMN_INDEX|WITH_WEIGHT Users tags
```
Comment (kou) [inline test/command/suite/select/scorer/expression/multiplication.test]: Do we need index?

File: test/command/suite/select/scorer/expression/multiplication.test [hunk 2]
```diff
@@ -0,0 +1,15 @@
+table_create Users TABLE_HASH_KEY ShortText
+column_create Users tags COLUMN_VECTOR|WITH_WEIGHT ShortText
```
Comment (kou) [inline test/command/suite/select/scorer/expression/multiplication.test]: Do we need `WITH_WEIGHT`?

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed them.

**Comment by kou** [comment]
We should fix this ASAP.

**Pull request merged**